py/desitarget/cuts.py

```diff
@@ -740,6 +750,9 @@ def isSTD(gflux=None, rflux=None, zflux=None, primary=None,
            logic cuts. If Gaia is not available (perhaps if you're using mocks)
            then send ``False`` and pass `gaiagmag` as 22.5-2.5*np.log10(`robs`) 
            where `robs` is `rflux` without a correction.for Galactic extinction.
+        south: boolean, defaults to True
```

Comment:
Please modify the documentation to indicate that it is not necessary to pass an r flux if `usegaia=False`